test/e2e-tests.sh

```diff
@@ -90,11 +90,17 @@ sleep 30
 
 # Run conformance and e2e tests.
 
+# Currently only Istio and Contour implement the alpha features.
+alpha=""
+if [[ -z "${INGRESS_CLASS}" || "${INGRESS_CLASS}" == "istio.ingress.networking.knative.dev" || "${INGRESS_CLASS}" == "contour.ingress.networking.knative.dev" ]]; then
```

Comment:
D'oh! [Excluded in the GitHub action](https://github.com/knative/serving/blob/master/.github/workflows/kind-e2e.yaml#L57) but not here - thanks! Let's add a link to https://github.com/knative/serving/issues/9874 like we do in the GitHub action version so we remember to unexclude it?

Reply:
Done!